.github/workflows/build_docker.yml

```
@@ -62,7 +62,26 @@ jobs:
         DOCKER_ORGANIZATION: ${{ secrets.DOCKER_ORGANIZATION }}
         GITHUB_ORGANIZATION: ${{ secrets.GITHUB_ORGANIZATION }}
 
-  build_blackduck_with_dotnetcore:
+  build_blackduck_with_dotnetcore_2_2_110:
+    name: Build blackduck with dotnet core 2.2.110
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - uses: actions/checkout@master
+      with:
+        submodules: true
+
+    - name: Build BlackDuck Docker Images with dotnet core 2.2.110
+      run: |
+        ./ci/bin/docker_build_and_push.sh 6/dotnetcore-2.2.110 blackduck:dotnetcore-2.2.110 blackduck:6-dotnetcore-2.2 blackduck:6.0-dotnetcore-2.2.110 blackduck:6.0.0-dotnetcore-2.2.110
```

```suggestion
        ./ci/bin/docker_build_and_push.sh 6/dotnetcore-2.2.110 blackduck:dotnetcore-2 blackduck:6-dotnetcore-2 blackduck:6.0-dotnetcore-2.2 blackduck:6.0.0-dotnetcore-2.2.110
```
I need to write down how my mind is thinking about the versioning of the docker images. I think it's up for discussion, but for me this feels more logical.

---

Thanks for the catch. Updated README. Please have a look.
Regarding versioning we can have a discussion.